.travis.yml

```diff
@@ -7,6 +7,7 @@ cache:
   - depends/built
   - depends/sdk-sources
   - $HOME/.ccache
+  - build/releases
```

Comment:
That cache should be covered by `.ccache`, no?

Reply:
- [ ] will look into `ccache` behavior.

Reply:
Forget what I said. It is probably better to cache the resulting binary than the individual obj files, since we are building specific tags that don't change. For master or other branch build, the commits often only change parts of the code, so the other obj files can be cached and re-used. For tag builds everything is static, so caching the binaries works as well.